services/api-server/requirements/_test.txt

```diff
@@ -22,61 +22,61 @@ cffi==1.14.2              # via -r requirements/_base.txt, bcrypt, cryptography,
 change-case==0.5.2        # via -r requirements/_test.in
 chardet==3.0.4            # via -r requirements/_base.txt, aiohttp, httpx, requests
 click==7.1.2              # via -r requirements/_base.txt, uvicorn
-codecov==2.1.8            # via -r requirements/_test.in
+codecov==2.1.9            # via -r requirements/_test.in
 contextvars==2.4          # via -r requirements/_base.txt, sniffio
 coverage==5.2.1           # via codecov, coveralls, pytest-cov
 coveralls==2.1.2          # via -r requirements/_test.in
-cryptography==3.0         # via -r requirements/_base.txt, paramiko
+cryptography==3.1         # via -r requirements/_base.txt, paramiko
 dataclasses==0.7          # via -r requirements/_base.txt, pydantic
 distro==1.5.0             # via docker-compose
 dnspython==2.0.0          # via -r requirements/_base.txt, email-validator
-docker-compose==1.26.2    # via pytest-docker
-docker[ssh]==4.3.0        # via -r requirements/_test.in, docker-compose
+docker-compose==1.26.2    # via -r requirements/_test.in, pytest-docker
```

Comment:
we should prevent docker-compose from being in here.
besides, this is only for the test requirements. what if we want to upgrade docker-compose and we cannot do it because of pytest-docker?

Reply:
check comment above